consensus/ibft/extra.go

```diff
@@ -120,6 +184,23 @@ func (i *IstanbulExtra) MarshalRLPWith(ar *fastrlp.Arena) *fastrlp.Value {
 		vv.Set(committed)
 	}
 
+	// ParentCommittedSeal
+	if i.ParentCommittedSeal != nil {
+		if len(i.ParentCommittedSeal) == 0 {
+			vv.Set(ar.NewNullArray())
+		} else {
+			committed := ar.NewArray()
+			for _, a := range i.ParentCommittedSeal {
+				if len(a) == 0 {
+					vv.Set(ar.NewNull())
```

Comment:
I don't see that ever happening, but if it does unmarshal would not work or i am missing something?
When we are doing unmarshaling for ParentCommittedSeal we are just extracting the 4th element with elems[3].GetElems and If we had multiple PAS which has len(a)==0 we would have more than 4 elements.
Please correct me if i understood something wrong :)